lib/graphql/language/nodes.rb

```diff
@@ -202,7 +202,7 @@ class FragmentDefinition < AbstractNode
         def initialize_node(name: nil, type: nil, directives: [], selections: [])
           @name = name
           @type = type
-          @directives = directives
+          @directives = directives || []
```

Comment:
I see that directives is often falling back to `|| []` -- what's the case where this is needed? (I ask because I'm only aware of `directives_list_opt` in the parser, which already provides `[]` for the empty case.)